package.json

```diff
@@ -130,7 +130,9 @@
         "eslint-plugin-storybook": "0.6.2",
         "file-loader": "6.2.0",
         "husky": "8.0.3",
-        "jest": "26.6.3",
+        "jest": "29.6.1",
+        "jest-cli": "^29.6.1",
```

Comment:
Don't use `^`, it's better to specify exact versions

Reply:
:+1: